src/components/billing/info/infobar.styles.ts

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from "@mui/styles";
+
+const InfoBarStyles = makeStyles({
+  container: {
+    position: "relative",
+    display: "flex",
+    width: "928px",
+    height: "150px",
+    top: "50px",
+    borderRadius: "20px",
+    background: "#FFFFFF",
+  },
+  containerItem: {
+    position: "relative",
+    dispay: "flex !important",
+    width: "210px",
+    height: "82px",
+    top: "40px",
+    left: "39px",
+    borderRadius: "16px",
+  },
+  line: {
+    position: "absolute",
+    width: "82px",
+    top: "34px",
+    left: "125px",
+    transform: "rotate(-90deg)",
+    background: "#ECECEE",
+    border: "1px solid #ECECEE",
+  },
+  iconContainer: {
+    width: "63px",
+    height: "63px",
+    top: "211px",
+    left: "50px",
```

Comment:
Not a good way wo css. It will be hard to make this dynamic if we move to mobile version.